setup.py

```diff
@@ -9,16 +9,9 @@
     install_requires=[
         'attrs>=17.1.0',
         'structlog',
+        'aiohttp>=2',
```

Comment:
Yes. Talk to the webdriver server (eg geckodriver) happens over http, so we need an async http client such as aiohttp.